dm/master/openapi.go

```diff
@@ -215,12 +222,96 @@ func (s *Server) DMAPIGetSourceStatus(ctx echo.Context, sourceName string) error
 
 // DMAPIStartTask url is:(POST /api/v1/tasks).
 func (s *Server) DMAPIStartTask(ctx echo.Context) error {
-	return nil
+	var req openapi.CreateTaskRequest
+	if err := ctx.Bind(&req); err != nil {
+		return err
+	}
+	task := req.Task
+	if err := adjustModelTask(&task); err != nil {
+		return err
+	}
+	// prepare target db config
+	newCtx := ctx.Request().Context()
+	toDBCfg, err := getAndAdjustTaskTargetDBCfg(newCtx, &task)
```

Comment:
also slightly recommend make it `(*Task).GetTargetDBCfg`

and now `(*Task).GetTaskConfig` is more acceptable.

Reply:
added in https://github.com/pingcap/dm/pull/2141/commits/268f6007a30f039603b5699bed6fedf578ec175f